### PR TITLE
Support HTTP POST with JSON parameters in imageREST_v1

### PR DESCRIPTION
**Pull request by kennywlo**

### CODE REVIEW

File: python/lsst/dax/imgserv/getimage/imagegetter_v1.py [hunk 1]
```diff
@@ -451,16 +451,17 @@ def _apply_cutout(self, src_img, metadata, ra, dec, width, height, data_id,
     def _metadata_from_data_id(self, data_id):
         # Return the metadata for the query results in qResults and a butler.
```
Comment (brianv0) [inline python/lsst/dax/imgserv/getimage/imagegetter_v1.py]: I think we might want to defensively program and add a `metadata = None` in the case `self._butler_keys` is neither of the two options? Or, can we turn the statements to an `if`, `elif`, and an `else: raise SomeException`?

File: python/lsst/dax/imgserv/imageREST_v1.py [hunk 2]
```diff
@@ -154,19 +158,37 @@ def _getimage(_req, db_id):
         request the request object
         db  image database string
     """
-    ds = _req.args.get("ds")
+    if _req.method == "POST":
+        # check  mimetype
+        mime_type =_req.headers["Content-Type"]
```
Comment (brianv0) [inline python/lsst/dax/imgserv/imageREST_v1.py]: I think you should check before here if the input is actually a form too, in which case, treat the POST request the same as the GET request in line 178. I think that should work just fine. We can add a ticket to accept this improvement later too.

File: python/lsst/dax/imgserv/cli_v1.py [hunk 3]
```diff
@@ -157,25 +172,20 @@ def _get_ds(self, req):
...
+                    schema = json.load(s)
+                    s.close()
+                validate(req, schema)
         except ValidationError as e:
-            msg = json.loads({"Validation Error": e.message})
+            msg = json.dumps({"Validation Error": e.message})
             return 1, msg
-        # read in the request (JSON)
-        req = json.loads(req_data)
         return 0, req
```
Comment (brianv0) [inline python/lsst/dax/imgserv/cli_v1.py]: Too much whitespace at end of file, can you clean that up? (1 line after last line of code)

File: python/lsst/dax/imgserv/getimage/imagegetter_v1.py (hunk 1 above)
Comment (kennywlo) [reply python/lsst/dax/imgserv/getimage/imagegetter_v1.py]: Added metadata = None using if, elif, and else construct.  The method may return None, which caller will have to check for.

File: python/lsst/dax/imgserv/cli_v1.py (hunk 3 above)
Comment (kennywlo) [reply python/lsst/dax/imgserv/cli_v1.py]: Deleted blank space as suggested.

File: python/lsst/dax/imgserv/imageREST_v1.py (hunk 2 above)
Comment (kennywlo) [reply python/lsst/dax/imgserv/imageREST_v1.py]: I've implemented support for form post here as well, resulting in much simpler the logic flow.

**Pull request merged**